connectors/src/connectors/confluence/temporal/workflows.ts

```diff
@@ -0,0 +1,121 @@
+import type { ModelId } from "@dust-tt/types";
+import {
+  executeChild,
+  proxyActivities,
+  workflowInfo,
+} from "@temporalio/workflow";
+
+import type * as activities from "@connectors/connectors/confluence/temporal/activities";
+import type { DataSourceConfig } from "@connectors/types/data_source_config";
+
+// The Temporal bundle does not support the use of aliases in import statements.
+import { makeConfluenceSpaceSyncWorkflowIdFromParentId } from "./utils";
+
+const {
+  confluenceGetSpaceNameActivity,
+  confluenceListPageIdsInSpaceActivity,
+  confluenceSaveStartSyncActivity,
+  confluenceUpsertPageActivity,
+  confluenceSaveSuccessSyncActivity,
+
+  fetchConfluenceConfigurationActivity,
+  getSpaceIdsToSyncActivity,
+} = proxyActivities<typeof activities>({
+  startToCloseTimeout: "20 minutes",
+});
+
+export async function confluenceFullSyncWorkflow({
+  connectionId,
+  connectorId,
+  dataSourceConfig,
+  spaceIdsToBrowse,
+}: {
+  connectionId: string;
```

Comment:
Remove connectionId and dataSourceConfig.

Reply:
Will do in a follow up PR since this code is not yet used in production. I'll create an issue to patch the other connectors as well.